test/cli/cli_output_sketches.cpp

```diff
@@ -0,0 +1,55 @@
+// ---------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2023, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2023, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/chopper/blob/main/LICENSE.md
+// ---------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <filesystem>
+#include <fstream>
+#include <string> // strings
+
+#include <seqan3/test/tmp_directory.hpp>
+
+#include "cli_test.hpp"
+
+TEST_F(cli_test, chopper_layout)
+{
+    seqan3::test::tmp_directory tmp_dir{};
+    std::filesystem::path const input_filename{tmp_dir.path() / "data.filenames"};
+    std::filesystem::path const layout_filename{tmp_dir.path() / "output.binning"};
+    std::filesystem::path const sketches_dir{tmp_dir.path() / "sketches"};
```

Comment:
The `cli_test` fixture already takes care of creating a temporary worling directory, so you can just use, e.g., `"data.filenames"` as path.